site-config/hca-dcp/cc-ma-dev/index/filesEntityConfig.ts

```diff
@@ -0,0 +1,23 @@
+import { EntityConfig } from "@databiosphere/findable-ui/lib/config/entities";
+import { COLUMN } from "./common/constants";
+
+/**
+ * Returns the entity config for managed access files.
+ * @param entitiesConfig - Entities config.
+ * @returns entity config for managed access files.
+ */
+export function getMAFilesEntityConfig(
+  entitiesConfig: EntityConfig
+): EntityConfig {
+  // Clone files entity.
+  const cloneEntity = { ...entitiesConfig };
+  // Clone list.
+  const cloneList = { ...cloneEntity.list };
+  // Clone columns.
+  const cloneColumns = [...cloneList.columns];
+  // Add data use restriction column.
+  cloneColumns.splice(2, 0, COLUMN.DATA_USE_RESTRICTION);
```

Comment:
The shape returned for `files`'s `project` is different to `projects`'s `project` response. e.g.

```
dataUseRestriction: [null]
```

1. I believe you will need to add to [AggregatedProjectResponse](https://github.com/DataBiosphere/data-browser/blob/c4335416e8d0f456e58f15d8419df5ab616ec309/app/apis/azul/hca-dcp/common/aggregatedEntities.ts#L80) `dataUseRestriction: (null | string)[]`.
2. On line 19, remove the reference `COLUMN.DATA_USE_RESTRICTION` because this is `projects` response related. I think for now let's just insert the new config like this:

```
  cloneColumns.splice(2, 0, {
    componentConfig: {
      component: C.BasicCell,
      viewBuilder: V.buildDataUseRestrictions,
    } as ComponentConfig<typeof C.BasicCell, FilesResponse>,
    header: HCA_DCP_CATEGORY_LABEL.DATA_USE_RESTRICTION,
    id: HCA_DCP_CATEGORY_KEY.DATA_USE_RESTRICTION,
    width: { max: "1fr", min: "146px" },
  });
```

The `viewBuilder` `V.buildDataUseRestrictions` should probably look something like this:

```
export const buildDataUseRestrictions = (
  filesResponse: FilesResponse
): React.ComponentProps<typeof C.BasicCell> => {
  return {
    value: processEntityArrayValue(
      filesResponse.projects,
      "dataUseRestriction"
    ),
  };
};
```

Confirm on the `files` tab that you can see in the `Data Use Restriction` column, data such as "NRES" or "Unspecified".

<img width="1088" alt="image" src="https://github.com/user-attachments/assets/28648b32-ff74-4432-8dde-da1ee607cf99">